triggerbot/tree_watcher.py

```diff
@@ -57,8 +58,10 @@ def __init__(self, ldap_auth, is_triggerbot_user=lambda _: True):
         self.auth = ldap_auth
         self.lower_trigger_limit = TreeWatcher.default_retry * TreeWatcher.per_push_failures
         self.log = logging.getLogger('trigger-bot')
+        logging.basicConfig()
         self.is_triggerbot_user = is_triggerbot_user
         self.global_trigger_count = 0
+        self.treeherderapi = TreeherderApi()
```

Comment:
We use this to get data for all jobs of a repository and request ids for them, so does "treeherder_repo" sounds like a good name?
So, it will be: self.treeherder_repo.get_all_jobs(branch, rev) , self.treeherder_repo.get_matching_jobs, self.treeherder_repo.get_buildapi_request_id(branch, job)